README.md

```diff
@@ -1,16 +1,23 @@
-# jspecify
+# jspecify 0.1
 
-**This project is a placeholder.** We are using the issue tracker for some of our conversations, but the existing code in the repo does not reflect our current design decisions, and we have many decisions left to make.
+(This README is for the 0.1 milestone. For the main project README, see
+[our GitHub landing page](https://github.com/jspecify/jspecify).)
 
-An artifact of well-named and well-specified annotations to power static
-analysis checks.
+This milestone will be tagged around August 19, 2020.
```

Comment:
This sentence will look slightly awkward once tagging actually happened. Not an issue if one remembers to update it just prior to tagging, but safer might be to leave it out.

(What I guess we should do (post-tagging) is to link to the tag from the `main` README and have a sentence along these lines there. Something like "On August 19, 2020 we tagged release 0.1".)

Reply:
Makes sense. And it occurs to me only now that GitHub will produce a list of releases for us, anyway.